website/views.py

```diff
@@ -88,30 +88,36 @@ def _render_nodes(nodes, auth=None, show_path=False, parent_node=None):
 
 
 def index():
-    try:
-        #TODO : make this way more robust
-        institution = Institution.find_one(Q('domains', 'eq', request.host.lower()))
-        inst_dict = view_institution(institution._id)
-        inst_dict.update({
-            'home': False,
-            'institution': True,
-            'redirect_url': '/institutions/{}/'.format(institution._id)
-        })
-
-        return inst_dict
-    except NoResultsFound:
-        pass
-
-    all_institutions = Institution.find().sort('name')
-    dashboard_institutions = [
-        {'id': inst._id, 'name': inst.name, 'logo_path': inst.logo_path_rounded_corners}
-        for inst in all_institutions
-    ]
-
-    return {
-        'home': True,
-        'dashboard_institutions': dashboard_institutions
-    }
```

Comment:
This looks like a large change, but all I added was a check for a logged in user and indented everything.